docs/source/logos.rst

```diff
@@ -3,29 +3,35 @@
 Images and Logos
 ================
 
+Here are some commonly used Dask icons and logos
+(see the `Dask style guide <https://www.dask.org/style-guide>`_ for more details).
+
 .. image:: images/dask_icon.svg
-   :alt: Dask logo.
+   :alt: Primary Dask icon.
+
+.. image:: images/dask_icon_black.svg
+   :alt: Dask icon in black.
 
-.. image:: images/dask_icon_no_pad.svg
-   :alt: Dask logo without padding.
+.. image:: images/dask_icon_white.svg
+   :alt: Dask icon in white.
+
+.. image:: images/dask_icon_on_pink.svg
+   :alt: Dask icon to use on a pink background.
 
 .. image:: images/dask_horizontal.svg
-   :alt: Dask logo.
+   :alt: Primary Dask logo.
 
-.. image:: images/dask_horizontal_no_pad.svg
-   :alt: Dask logo without padding.
+.. image:: images/dask_horizontal_black.svg
+   :alt: Dask logo in black.
 
 .. image:: images/dask_horizontal_white.svg
-   :alt: Dask logo.
-
-.. image:: images/dask_horizontal_white_no_pad.svg
-   :alt: Dask logo.
+   :alt: Dask logo in white.
 
-.. image:: images/dask_stacked.svg
-   :alt: Dask logo.
+.. image:: images/dask_horizontal_on_pink.svg
+   :alt: Dask logo to use on a pink background.
 
-.. image:: images/dask_stacked_white.svg
-   :alt: Dask logo.
+.. image:: images/dask_horizontal_on_blue.svg
+   :alt: Dask logo to use on a blue background.
```

Comment:
This is kind of unrelated, but the janelia logo below seems out of place. I think we should just remove it from this page.

Reply:
I think keeping it in the assets and just not having it on this page makes sense.